### ISSUE TITLE
`TestTfmsInParallel: false` setting is not works after update to xUnit v3

### ISSUE THREAD

**Issue by filzrev** [xunit/visualstudio.xunit#429]
I'm using `<TestTfmsInParallel>false</TestTfmsInParallel>` setting to disable parallel execution between TargetFrameworks.
But after updating to xUnit v3.
It seems not works as expected.

**Reproduce steps**
1. Create xUnit v2 based project. And update packages to latest version (xUnit: 2.9.2/ xunit.runner.visualstudio: 3.0.0)
2. Modify .csproj with following changes. 
  2.1. Set `<TargetFrameworks>net8.0;net9.0<TargetFrameworks>`
  2.2. Set `<TestTfmsInParallel>false</TestTfmsInParallel>`
3. Insert `Thread.Sleep(5000)` to unit test code.
4. Run tests on Test Explorer and confirms tests is isolated between target frameworks.
5. Modify .csproj to use xUnit v3 (`<PackageReference Include="xunit.v3" Version="1.0.0" />`)
6. Run tests on Test Explorer and confirms tests is executed in parallel.

**Environment**
I've confirmed it can be reproduced on following environment.
- Visual Studio 2022 Version 17.12.0
- Visual Studio 2022 Version 17.13.0 Preview 2.0

**Comment by filzrev** [comment]
I've confirmed same issue can be reproduced when using `MSTest.Sdk/3.6.3`.
So it this issue might be caused by `Microsoft.Testing.Platform` side

**Comment by bradwilson** [comment]
I agree, especially because I had no idea `TestTfmsInParallel` even existed before today. 😂

**Comment by filzrev** [comment]
As stated on https://github.com/microsoft/testfx/issues/4392#issuecomment-2557629554 comment. 
It seems be caused by VS Test Explorer side problems.

Currently it's tracked on following ticket.
https://developercommunity.visualstudio.com/t/TestTfmsInParallel:false-does-not-work-w/10817302

I'll close this issue when it's resolved on latest VS preview.

**Comment by bradwilson** [comment]
This was closed because it doesn't belong to us.

The Visual Studio issue that was opened was closed as "by design". I don't think this issue is getting fixed.

**Comment by filzrev** [comment]
This issue seems to be resolved on recent version of Visual Studio. (I've tested on `17.14.10`)

When `Run Tests in Parallel` option is not enabled. (Default Mode)
Tests are executed sequentially per project/tfm on TestExplorer.

<img width="335" height="241" alt="Image" src="https://github.com/user-attachments/assets/35b9adab-3bb8-4454-be3e-145edd792e24" />


## Excerpt of Developer Community comment  

> In Test Window you have a button to “Run tests in parallel” which is not enabled by default.
> This is how the old vstest.console.exe projects run their tests one at a time irrespective of the TestTfmsInParallel setting.

> The change I’ve made is to respect the “Run tests in parallel” Test Window option for Microsoft.Testing.Platform
> and not run tests in parallel by default. If the setting is enabled, then all projects and all tfm are run in parallel for both vstest.console.exe and Microsoft.Testing.Platform. I’ll know in a couple of days better which VS version will get the fix.
